playbooks/software-modules.yml

```diff
@@ -0,0 +1,12 @@
+#
+# install software modules using EasyBuild.
+#
+---
+- hosts: all
+  roles:
+    - name: lmod
+
+- hosts: easy-build-node
```

Comment:
Can we just set this to slurm-master and get rid of the easy-build-node group as a default?

Reply:
No, sometimes you want to build the modules on a GPU node. The slurm master may or may not have GPUs. that is why I think we should keep it.